server_go111.go

```diff
@@ -0,0 +1,43 @@
+// +build go1.11,!windows
```

Comment:
@miekg The full list of build tags taken from the `x/sys/unix` package is: `aix darwin dragonfly freebsd linux netbsd openbsd solaris`. I've gone with `!windows` for now, because it matches `udp.go`. This neglects `plan9` though, so I'm not sure what you'd prefer here: the full list of just not-windows.

Reply:
fine